src/components/shared/formItems/ProjectReconfiguration.tsx

```diff
@@ -0,0 +1,70 @@
+import { t, Trans } from '@lingui/macro'
+import { Form } from 'antd'
+import { CSSProperties, useContext, useState } from 'react'
+import { ThemeContext } from 'contexts/themeContext'
+
+import {
+  ballotStrategies,
+  Strategy,
+} from 'constants/ballotStrategies/ballotStrategies'
+import { getBallotStrategyByAddress } from 'constants/ballotStrategies/getBallotStrategiesByAddress'
+
+import ReconfigurationDrawer from '../ReconfigurationDrawer'
+import ReconfigurationOption from '../ReconfigurationOption'
+
+export default function ProjectReconfiguration({
+  value,
+  onChange,
+  style,
+}: {
+  value: string
+  onChange: (address: string) => void
+  style?: CSSProperties
+}) {
+  const {
+    theme: { colors },
+  } = useContext(ThemeContext)
+  const threeDayDelayStrategy = ballotStrategies()[2]
```

Comment:
**suggestion**: good practice to keep constants that don't rely on any state or props _outside_ the component function.